test/ecto/migrator_test.exs

```diff
@@ -45,6 +45,25 @@ defmodule Ecto.MigratorTest do
     end
   end
 
+  defmodule NoTransactionMigration do
+    use Ecto.Migration
+    @disable_ddl_transaction true
+
+    def change do
+      create index(:posts, [:foo])
+    end
+  end
+
+  defmodule TransactionMigration do
```

Comment:
Can we use an existing migration? Avoid defining many modules as they become hard to maintain and affect the testing time considerably. :)